app/src/main/java/com/example/dynamicdiet/service/DietService.kt

```diff
@@ -29,7 +29,7 @@ class DietService {
      * @param caloriesPerDay The number of calories per day set for the diet
      * @return newly updated Diet object
      */
-    suspend fun updateDiet(var userId: Int, var name: String, var caloriesPerDay: Int): Diet {
+    suspend fun updateDiet( userId: Int,  name: String,  caloriesPerDay: Int) { //removed var
```

Comment:
Removed var and : Diet this allows it to compile

https://kotlinlang.org/docs/coding-conventions.html#class-headers